baseplate/context/cassandra.py

```diff
@@ -3,12 +3,19 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import logging
+import sys
+from threading import Event
```

Comment:
note: this appears to be un-monkeypatched in gevent < 1.3 so we should make sure we're on 1.3+

Reply:
Should I import `Event` from `gevent` instead? Or do you mean like do an audit of all our services and ensure they're on 1.3+ or upgrade them before releasing this?

Reply:
I think we want to avoid importing from gevent in baseplate itself?  I'm not sure if we have a great way to enforce this though other than just doing an audit.

Reply:
I'm gonna play around with setup.py a bit, there might be a way to sanely say we need 1.3+ when we're using gevent without pulling it in as a hard dependency.

Reply:
Should I merge this now, or hold off until we figure this stuff out?

Reply:
let's merge. i've no good answers on versions right now.